tests/lib/modem_trace/modem_trace_rtt/src/modem_trace_test.c

```diff
@@ -26,14 +26,10 @@ extern int generic_suiteTearDown(int num_failures);
 
 void setUp(void)
 {
-	mock_SEGGER_RTT_Init();
```

Comment:
Why are these removed?

Reply:
I have never seen this in Unity tests. Seems to be an autogenerated mock, but did not understand what it's supposed to do, and it does not seem to actually do anything since the tests were passing without this.

Reply:
That's done automatically when using `_Expect` and `_AddCallback` according to the [documentation](https://github.com/ThrowTheSwitch/CMock/blob/master/docs/CMock_Summary.md).
`_Stub` does not do that; I have changed some `_AddCallback` to `_Stub`, so I will revert that change.
I will also re-introduce the `_Init` and `_Verify`, but out of curiosity, where did you find them?
I could not find any doc about them.

Reply:
This is because we are using custom test runner generator in sdk-nrf. And when we do that, these `_Init` and `_Verify` functions need to be called before and after each test. See https://github.com/ThrowTheSwitch/CMock/issues/202#issuecomment-468652635 

Here is the commit where I first introduced these functions in an ncs unit test case. I explain this in the commit message.

https://github.com/nrfconnect/sdk-nrf/commit/6268fbcedab20a66ff238b35ed24a7d55aa68aae

Reply:
Thank you for clarifying 👍🏻